src/django_management/settings.py

```diff
@@ -28,6 +28,7 @@
 ALLOWED_HOSTS = []
 
 # Application definition
+SITE_ID = 3
```

Comment:
Could you add a comment why this is set to `3`?

Reply:
Updated the code to automate the Google login setup. Reopen if required.